decidim-proposals/app/presenters/decidim/proposals/collaborative_draft_presenter.rb

```diff
@@ -29,20 +29,21 @@ def collaborative_draft_path
       end
 
       def title(links: false, extras: true, html_escape: false)
-        renderer = Decidim::ContentRenderers::HashtagRenderer.new(collaborative_draft.title)
-        text = renderer.render(links: links, extras: extras).html_safe
+        text = collaborative_draft.title
         text = decidim_html_escape(text) if html_escape
-        text
+
+        renderer = Decidim::ContentRenderers::HashtagRenderer.new(text)
+        renderer.render(links: links, extras: extras).html_safe
       end
 
       def body(links: false, extras: true, strip_tags: false)
-        renderer = Decidim::ContentRenderers::HashtagRenderer.new(collaborative_draft.body)
+        text = collaborative_draft.body
+        text = strip_tags(text) if strip_tags
+
+        renderer = Decidim::ContentRenderers::HashtagRenderer.new(text)
         text = renderer.render(links: links, extras: extras).html_safe
-        if strip_tags
-          text = strip_tags(text)
-          text = Anchored::Linker.auto_link(text, target: "_blank", rel: "noopener")
-        end
-        text
+
+        Anchored::Linker.auto_link(text, target: "_blank", rel: "noopener")
```

Comment:
Today I was thinking to open a new PR because  text links are always promoted to anchors. But there's already a `links` flag that we can reuse and make the behavior more consistent.

What's your opinion, do you want me to open a new PR with this change?

Reply:
```suggestion
        text = Anchored::Linker.auto_link(text, target: "_blank", rel: "noopener") if links
        text
```

Reply:
Yes, that makes sense @tramuntanal!